### ISSUE TITLE
[Adjust Rule] SIM903

### ISSUE THREAD

**Issue by MaxG87** [MartinThoma/flake8-simplify#130]
## Desired change

SIM903 should not be flagged in the following snippet:

```python
import datetime as dt

time = dt.datetime(2022, 3, 29, 12, 0, 0)
date = dt.date(2022, 3, 29)
```

## Explanation

In the given example, I would consider added keywords to be distracting boilerplate. I think in the given example it is obvious what the arguments mean.

**Comment by MartinThoma** [comment]
Uh, right. This rule has some other issues (positional-only arguments). I've already moved SIM902 to [flake8-scream](https://github.com/MartinThoma/flake8-scream) for this reason.

https://github.com/MartinThoma/flake8-simplify/pull/131 will be merged in 1-2 hours. I then also make a release where this rule is removed.

**Comment by MartinThoma** [comment]
[`flake8-simplify==0.19.2`](https://pypi.org/project/flake8-simplify/#history) was just released with the fix :-)
